### ISSUE TITLE
TypeScript import error: You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

### ISSUE THREAD

**Issue by Amaimersion** [wojtekmaj/react-calendar#224]
I'm using TypeScript and doing following import:
```typescript
import Calendar from "react-calendar";
...
render() {
  <Calendar />
}
```
And i get following error: `Uncaught Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.`

Here is a screenshot from developer console (don't pay attention to `react-loadable`) 
![1](https://user-images.githubusercontent.com/20613502/62416785-aba38900-b649-11e9-81e9-cc60e0788a0d.png)

For example, using of `MonthView` works fine. But entire default module not working.

TypeScript version – 3.5.3
React version – 16.8.6
React Calendar version – 2.19.0

**Comment by tejovanthWork** [comment]
Hey @Amaimersion, can you post some repo with this bug? I'm unable to reproduce this issue with the same versions.

**Comment by wojtekmaj** [comment]
Closed because of #273

**Comment by wojtekmaj** [comment]
FYI, React-Calendar is now natively in TypeScript, as of #785, coming to npm very soon.
